File: Java/1959.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);

        String[] line = in.readLine().trim().split(" ");
        
        System.out.println(Long.parseLong(line[0]) * Long.parseLong(line[1]));

    }
}

File: C#/1959.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI{
    static void Main(string[] args){
        List<long> line = Console.ReadLine().Trim().Split(' ').Select((x) => long.Parse(x)).ToList();
        
        Console.WriteLine(line[0] * line[1]);
    }
}